ci/test.ps1

```diff
@@ -29,8 +29,32 @@ function run_test {
 	$TestCommand = (ctest -N -V -R "^$TestName$") -join "`n" -replace "(?ms).*\n^[0-9]*: Test command: ","" -replace "\n.*",""
 	$TestCommand += " -r${BuildDir}\results_${TestName}.xml"
 
-	Invoke-Expression $TestCommand
-	if ($LastExitCode -ne 0) { $global:Success = $false }
```

Comment:
One thing I wondered about recently: do we really need the Powershell file at all? I saw that Azure does support Bash on Windows via Git for Windows. If that's the case, then we should drop this file completely so that we don't always have to keep both in sync. I'd naturally volunteer to do that.

Reply:
That's true, it does.  I have no love for PowerShell...  I'm happy to do that, too, but I've got a few other things going on before I can look at it.